examples/wearable/UIComponents/contents/navigationelements/index.html

```diff
@@ -40,6 +40,11 @@ <h2 class="ui-title">
 						SectionChanger
 					</a>
 				</li>
+				<li>
```

Comment:
coul you move this link up below IndexScroll so we can keep original order

Reply:
Done